src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Application {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[new Random().nextInt(10)];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
```

Comment:
In the main() method we need to create an array of figures (the size of array can be 3 or 6, it doesn't matter). The first half of figures in this array should be generated with random parameters.
The second half should cause a default figure.
try using one `loop`
```suggestion
            figureSupplier.getRandomFigure().draw();
```